int32 img_count
float32 class_id
float32 cx
float32 cy
float32 w
float32 h
